packages/generators/__tests__/init-generator.test.ts

```diff
@@ -1,7 +1,8 @@
 import { join } from 'path';
 import { run } from 'yeoman-test';
 
-describe('init generator', () => {
+// fixme: unstable
```

Comment:
Is this a flaky test? Seems to pass locally.

Reply:
Locally but on Travis it times out

Reply:
Maybe we can increase the timeout for these tests?

Reply:
Make a PR and let's see

Reply:
Sure thing 👍